Detections/WindowsEvent/ChiaCryptoMining_WindowsEvent.yaml

```diff
@@ -0,0 +1,64 @@
+id: 4d173248-439b-4741-8b37-f63ad0c896ae
```

Comment:
We use tags to refer to the original for code maintenance reasons. See [here](https://github.com/Azure/Azure-Sentinel/blob/master/Detections/ASimWebSession/PossibleDGAContacts.yaml)